website/docs/d/cloudinit_config.html.markdown

```diff
@@ -52,6 +52,15 @@ resource "aws_instance" "web" {
   instance_type = "t2.micro"
   user_data     = "${data.template_cloudinit_config.config.rendered}"
 }
+
+# Start a GCP instance with the cloudinit config as user data.
+resource "google_compute_instance" "web" {
+  name = "instance"
+  machine_type = "n1-standard-1"
+  metadata {
+    user-data = "${data.template_cloudinit_config.config.rendered}"      
```

Comment:
Showing a simple two-line startup script, either here or in a second example, might help people who are also just getting started with cloud-init.  Something like:

```
user-data = "#!/bin/bash\necho bing > /tmp/baz\n"
```